tests/grouped_values.rs

```diff
@@ -0,0 +1,190 @@
+mod utils;
+
+use clap::{App, AppSettings, Arg};
+
+#[test]
+fn value_sets_works() {
```

Comment:
Don't know why the test case names are starting with `value_sets`

Reply:
Changed to `grouped_value`